test/pie-alignTo.html

```diff
@@ -0,0 +1,154 @@
+                    // animation: false,
+                    labelLine: {
+                        minLength2: defaultMin,
+                        maxLength2: defaultMax
+                    },
+                    label: {
+                        edgeMargin: 20,
+                        position: 'outer',
+                        alignTo: 'labelLine',
+                        padding: 5
+                    }
+                }]
+            };
+
+            var option1 = {
+                series: [{
+                    type: 'pie',
+                    radius: '50%',
+                    data: data,
+                    // animation: false,
+                    labelLine: {
+                        minLength2: defaultMin,
+                        maxLength2: defaultMax
+                    },
+                    label: {
+                        edgeMargin: 20,
+                        position: 'outer',
+                        alignTo: 'edge'
+                    }
+                }]
+            };
+
+            var chart0 = testHelper.create(echarts, 'main0', {
+                title: 'alignTo: "labelLine"',
+                option: option0
+            });
+
+            var chart1 = testHelper.create(echarts, 'main1', {
+                title: 'alignTo: "edge"',
+                option: option1
+            });
+
+            var gui = new dat.GUI({
+            });
+            var config = {
+                maxLength2: defaultMax,
+                minLength2: defaultMin
+            };
+            var maxCtrl = gui
+                .add(config, 'maxLength2', 0, 1000)
+                .onChange(function (value) {
+                    if (value < config.minLength2) {
+                        config.minLength2 = value;
+                        option0.series[0].labelLine.minLength2 = value;
+                        option1.series[0].labelLine.minLength2 = value;
+                        minCtrl.setValue(value);
+                    }
+                    option0.series[0].labelLine.maxLength2 = value;
+                    option1.series[0].labelLine.maxLength2 = value;
+                    chart0.setOption(option0);
+                    chart1.setOption(option1);
+                });
+
+            var minCtrl = gui
+                .add(config, 'minLength2', 0, 1000)
+                .onChange(function (value) {
+                    if (value > config.maxLength2) {
+                        config.maxLength2 = value;
+                        option0.series[0].labelLine.maxLength2 = value;
+                        option1.series[0].labelLine.maxLength2 = value;
+                        maxCtrl.setValue(value);
+                    }
+                    option0.series[0].labelLine.minLength2 = value;
+                    option1.series[0].labelLine.minLength2 = value;
+                    chart0.setOption(option0);
+                    chart1.setOption(option1);
+                });
+            });
+        </script>
```

Comment:
Can you add another parameter to test `edgeMargin`? Also it will be great to record an interaction for testing.